docs/howto/set_database_schema.rst

```diff
@@ -0,0 +1,25 @@
+Set the database schema
+-----------------------
+
+Any PostgreSQL database contains one or more *schemas*. Schemas are namespaces really,
+not much more than that. See the `PostgreSQL Schema documentation`_ for more detail on
+schemas.
+
+By default Procrastinate uses the ``public`` schema, which is PostgreSQL's default
```

Comment:
> By default Procrastinate uses the ``public`` schema, which is PostgreSQL's default
schema

A bit tautological :)

Reply:
Ah, I misread sorry

Reply:
That being said... It's not 100% true, I think (not sure). Procrastinate doesn't care about its schema. I think the DB can be set so that the default schema is different, and then by default Procrastinate will use the default schema, no ?

Reply:
Not sure. You can change the `search_path` for a given session (using `SET search_path TO myschema`) but I am not sure you can attach a default schema to a database.

So, to my current knowledge, I think it's correct to say that Procrastinate will use the `public` schema by default.